src/routes/admin/wireless/createWirelessProfile.ts

```diff
@@ -0,0 +1,51 @@
+/*********************************************************************
+ * Copyright (c) Intel Corporation 2019
+ * SPDX-License-Identifier: Apache-2.0
+ * Author : Madhavi Losetty
+ **********************************************************************/
+import { validationResult } from 'express-validator'
+import { IWirelessProfilesDb } from '../../../repositories/interfaces/IWirelessProfilesDB'
+import { WirelessConfigDbFactory } from '../../../repositories/factories/WirelessConfigDbFactory'
+import { API_UNEXPECTED_EXCEPTION, API_RESPONSE } from '../../../utils/constants'
+import { WirelessConfig } from '../../../RCS.Config'
+import Logger from '../../../Logger'
+import { RPSError } from '../../../utils/RPSError'
+import { EnvReader } from '../../../utils/EnvReader'
+
+export async function createWirelessProfile (req, res): Promise<void> {
+  let profilesDb: IWirelessProfilesDb = null
+  let wirelessConfig: WirelessConfig = {} as WirelessConfig
+  const log = new Logger('createWirelessProfile')
+  try {
+    const errors = validationResult(req)
+    if (!errors.isEmpty()) {
+      res.status(400).json({ errors: errors.array() })
+      return
+    }
+    wirelessConfig = req.body
+    const passphrase = wirelessConfig.pskPassPhrase
+    if (req.secretsManager) {
+      wirelessConfig.pskPassPhrase = 'PSKPassPhrase'
+    }
+
+    profilesDb = WirelessConfigDbFactory.getConfigDb()
+    const results: WirelessConfig = await profilesDb.insertProfile(wirelessConfig)
+    if (results != null) {
```

Comment:
avoid indention if we can, flip this such that `if (results == null) { return }`

Reply:
done